index.cjs

```diff
@@ -747,19 +774,30 @@ safeql: try {
   }
 
   // Abort early if either of these modules are not installed
-  require.resolve('@ts-safeql/eslint-plugin');
-  require.resolve('dotenv-safe');
+  try {
+    require.resolve('@ts-safeql/eslint-plugin');
+    require.resolve('dotenv-safe');
+  } catch (error) {
+    throw new AddLineBreaksForEslintOutputInVscodeError(`Please reinstall the UpLeveled ESLint Config using the instructions on https://www.npmjs.com/package/eslint-config-upleveled
+${error}
+    `);
+  }
 
   // @ts-ignore 2307 (module not found) -- The require.resolve() above will ensure that dotenv-safe is available before this line by throwing if it is not available
   require('dotenv-safe').config();
 
-  if (
-    !process.env.PGHOST ||
-    !process.env.PGUSERNAME ||
-    !process.env.PGPASSWORD ||
-    !process.env.PGDATABASE
-  ) {
-    throw new Error('Environment variables are not set');
+  const requiredEnvVars = ['PGHOST', 'PGUSERNAME', 'PGPASSWORD', 'PGDATABASE'];
+
+  const missingEnvVars = requiredEnvVars.filter(
+    (envVar) => !process.env[envVar],
+  );
+
+  if (missingEnvVars.length > 0) {
+    throw new AddLineBreaksForEslintOutputInVscodeError(
+      `The following environment variables are not set: ${missingEnvVars.join(
+        ', ',
+      )}`,
+    );
```

Comment:
These changes improve the error reporting by dynamically identifying which specific environment variables are not set. The updated error message now lists the missing variables making it easier to identify the issue